<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="SplineEditorWidget" name="splineEditor"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="rightPanelVerticalLayout">
        <item>
         <widget class="QGroupBox" name="selectPointGroupBox">
          <property name="minimumSize">
           <size>
            <width>230</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>400</height>
           </size>
          </property>
          <property name="title">
           <string>Select Point</string>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>221</width>
             <height>181</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::ExpandingFieldsGrow</enum>
              </property>
              <property name="rowWrapPolicy">
               <enum>QFormLayout::DontWrapRows</enum>
              </property>
              <property name="horizontalSpacing">
               <number>6</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <item row="2" column="0">
               <widget class="QLabel" name="label_bias">
                <property name="text">
                 <string>Bias:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_bias">
                <item>
                 <widget class="QLineEdit" name="biasLineEdit">
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maxLength">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="biasHorizontalSlider">
                  <property name="focusPolicy">
                   <enum>Qt::StrongFocus</enum>
                  </property>
                  <property name="minimum">
                   <number>-10</number>
                  </property>
                  <property name="maximum">
                   <number>10</number>
                  </property>
                  <property name="pageStep">
                   <number>1</number>
                  </property>
                  <property name="value">
                   <number>0</number>
                  </property>
                  <property name="sliderPosition">
                   <number>0</number>
                  </property>
                  <property name="tracking">
                   <bool>true</bool>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="tickInterval">
                   <number>0</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_continuity">
                <property name="text">
                 <string>Continuity:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_continuity">
                <item>
                 <widget class="QLineEdit" name="continuityLineEdit">
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maxLength">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="continuityHorizontalSlider">
                  <property name="minimum">
                   <number>-10</number>
                  </property>
                  <property name="maximum">
                   <number>10</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_tension">
                <property name="text">
                 <string>Tension:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_tension">
                <item>
                 <widget class="QLineEdit" name="tensionLineEdit">
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maxLength">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="tensionHorizontalSlider">
                  <property name="minimum">
                   <number>-10</number>
                  </property>
                  <property name="maximum">
                   <number>10</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_position">
                <property name="text">
                 <string>Position:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_position">
                <item>
                 <widget class="QLabel" name="positionXLabel">
                  <property name="text">
                   <string>X</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="positionXLineEdit">
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>20</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="positionYLabel">
                  <property name="text">
                   <string>Y</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="positionYLineEdit">
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>20</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="DeletePushButton">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="AddPushButton">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="linesParamsGroupBox">
          <property name="minimumSize">
           <size>
            <width>230</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>400</height>
           </size>
          </property>
          <property name="title">
           <string>Lines params</string>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>8</x>
             <y>30</y>
             <width>211</width>
             <height>23</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QFormLayout" name="formLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Partitions: </string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSlider" name="partitionsNumberHorizontalSlider">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>30</number>
                </property>
                <property name="value">
                 <number>15</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="rightPanelVerticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>&amp;SaveAs</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SplineEditorWidget</class>
   <extends>QGraphicsView</extends>
   <header>splineeditorwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
